### DISCUSSION TITLE
"Last used" column on password settings page

### DISCUSSION THREAD

**Discussion by lukecarr** [planetscale/discussion#322] [Feature Requests] [answered]
It would be super useful to have a "Last used" column on the password settings page so you can easily identify and revoke inactive credentials.

Ideally, the passwords table would be sorted by this new column, instead of the creation timestamp.

**Comment by orware** [comment, marked as answer]
@lukecarr, I did share your thoughts internally after reviewing your comments here and this is an addition that should eventually be available within the platform!

Some pieces that are necessary for it are already in progress of being added in, but I don't have an exact date as to when it might be available. 

It is however definitely a metric that will have some nice utility once we can collect / report on it properly for credentials so thank you for asking about it!

**Reply by lukecarr** [reply]
Super, appreciate the transparency and look forward to it eventually being introduced!
